scss/components/input-group.scss

```diff
@@ -217,4 +223,9 @@ $block: #{$fd-namespace}-input-group;
             z-index: map-get($fd-z-index-levels, "second");
         }
     }
+    .fd-input {
+        &:focus {
+          box-shadow: none;
```

Comment:
So, is this the new focus styling for inputs in input groups (or just inputs in general)?

Reply:
@greg-a-smith This fixes the difference in styling pointed out in this comment https://github.com/SAP/fundamental-styles/pull/77#pullrequestreview-250640425. Also, represent the latest design. 

As for its application, I reached out to @LeoT7508 to have more details. For the mean time, I can use a more specific class to avoid bleeding while getting a response from the design team.